doc/api/url.md

```diff
@@ -470,6 +470,12 @@ the URL, use the [`url.search`][] setter. See [`URLSearchParams`][]
 documentation for details.
 
 #### `url.username`
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/31450
+    description: When used with `http.request()`, this field will now be
+                 percent-decoded.
```

Comment:
Missing closing `-->`.

Reply:
The section on `http.request()` in http.md seems like a more logical place to me for this note.

Reply:
> The section on `http.request()` in http.md seems like a more logical place to me for this note.

![Why not both? :)](https://user-images.githubusercontent.com/899444/73025759-745fa900-3e30-11ea-84e2-dad35b2f2808.jpg)